moped-editor/src/views/projects/projectView/ProjectComponents/utils/makeComponentData.js

```diff
@@ -147,6 +162,41 @@ export const getFeatureChangesFromComponentForm = (
       // signal was previously blank
       signalToCreate = knackSignalRecordToFeatureSignalsRecord(signalFromForm);
       signalToCreate.component_id = clickedComponent.project_component_id;
+      // if there was a beacon that was switched to signal, we need to clear that beacon
```

Comment:
This is also a good reminder of how we handle all of the update scenarios that can happen through the form!